packages/markdown-common/src/ToMarkdownStringVisitor.js

```diff
@@ -63,11 +64,12 @@ class ToMarkdownStringVisitor {
      * @param {*} parametersOut - the current parameters
      * @return {*} the new parameters with first set to true
```

Comment:
should this documentation now say `set to false` instead?

Reply:
oh yes. let me fix the comment. That comment didn't make much sense anyway since that was only used for block quotes and not generally working.